test/evergreen.yml

```diff
@@ -1521,6 +1584,7 @@ buildvariants:
     - name: make-check-asan-test
     - name: configure-combinations
     - name: checkpoint-filetypes-test
+    - name: coverage-test
```

Comment:
Prefer to retain the original name `test-coverage`

Reply:
It can be confusing with the convention that we put test at the end. What about coverage-report

Reply:
Basically, this task is to run some tests to generate a test coverage report. `coverage-report` sounds better than `coverage-test` or `test-coverage-test`.